api/pom.xml

```diff
@@ -58,7 +58,9 @@
                     <protocArtifact>com.google.protobuf:protoc:${protobuf.version}:exe:${os.detected.classifier}</protocArtifact>
                     <pluginId>grpc-java</pluginId>
                     <pluginArtifact>io.grpc:protoc-gen-grpc-java:${grpc.version}:exe:${os.detected.classifier}</pluginArtifact>
-
+                    <excludes>
+                        <exclude>**/v3alpha/*</exclude>
```

Comment:
do we still need this?

Reply:
Yes, this is needed as long as this folder exists: https://github.com/envoyproxy/envoy/tree/706f910954574e5d18dd4ba94ce66e8ee77b238c/api/envoy/api/v3alpha

Reply:
No, not needed. Removed.